sale_purchase_requisition/models/sale_order.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2023 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+
+PURCH_REQ_FROZEN_STATES = {"in_progress", "open", "done", "cancel"}
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    purchase_requisition_ids = fields.One2many(
+        comodel_name="purchase.requisition",
+        inverse_name="sale_id",
+        string="Purchase requisition",
+    )
+    opened_purchase_requisition_count = fields.Integer(
+        compute="_compute_opened_purchase_requisition_count",
+        help="Number of purchase requisitions in open state",
+        string="Opened Purchase Requisitions",
+    )
```

Comment:
[English is weird](https://dictionary.cambridge.org/pt/gramatica/gramatica-britanica/open-or-opened):

```suggestion
    open_purchase_requisition_count = fields.Integer(
        compute="_compute_opened_purchase_requisition_count",
        help="Number of purchase requisitions in open state",
        string="Open Purchase Requisitions",
    )
```